test/integration/fixtures/regression/issue-2286.js

```diff
@@ -0,0 +1,5 @@
+describe('suite', function () {
+  beforeEach(function () { this.skip() })
+  after(function () { console.log('after in suite') })
+  it('test', function () {})
+})
```

Comment:
@ScottFreeCode I don't see this stuff as absolutely necessary given the filename, but if you wish to add to the branch, go for it